src/Culling/babylon.boundingInfo.ts

```diff
@@ -146,22 +146,33 @@ module BABYLON {
         }
 
         /**
-         * Returns `true` if the bounding info is within the frustum defined by the passed array of planes.
-         * @param frustumPlanes defines the frustum to test
-         * @param strategy defines the strategy to use for the culling (default is BABYLON.Scene.CULLINGSTRATEGY_STANDARD)
-         * @returns true if the bounding info is in the frustum planes
+         * Returns `true` if the bounding info is within the frustum defined by the passed array of planes.  
+         * @param frustumPlanes defines the frustum to test  
+         * @param strategy defines the strategy to use for the culling (default is BABYLON.Scene.CULLINGSTRATEGY_BOUNDINGSPHERE_ONLY)  

```

Comment:
This is a change (unless there is no visual impact and only improvements?)

Reply:
only improvements, everythings runs like it used to... sometimes faster :-)

Reply:
But it is less accurate so you can end up having objects selected whereas they would have not with the regular strategy
So I would prefer to stick with the regular one as the default

Reply:
ooopps... actually nothing is changed in the process nor in the algorithm.
 The error is only in the comments : wrong copy/paste. 
The real default behavior is still the standard strategy.

I'll also change the comments about the static vars CULLINGSTRATEGY because there's actually no perceptible loss at all when using the Optimistic modes, only some possible little gains when including immediately.